docs/topics/Threads.md

```diff
@@ -0,0 +1,109 @@
+# Threads
+
+[Threads](#DOCS_RESOURCES_CHANNEL/channel-object) are a new Discord feature, only available in API v9. Bots that do not update to API v9 will not receive gateway events for threads, or things that happen in threads (such as [Message Create](#DOCS_TOPICS_GATEWAY/message-create)).
+
+Threads have been designed to be very similar to [channel](#DOCS_RESOURCES_CHANNEL/channel-object) objects, and this topic aggregates all of the information about threads, which should all help to make migrating very straightforward.
+
+## Disclaimer
+
+Threads have not shipped yet, and so everything in this documentation is still subject to change. At a minimum additional status codes will be added for reaching certain limits, but additional changes are still fair game, and likely.
```

Comment:
this wording kinda makes me worried. when is the latest we can expect to see breaking changes? because i imagine not all libraries might want to support them until then, and bots will still need some time to update to the lib update after that.

Reply:
after seeing that I had that thought too...  library maintainers would probably rather have a basically stable API instead of one that could change and break at whim

Reply:
I think _breaking_ changes are fairly unlikely at this point, the protocols are generally finished, and been tested for a few months now.  If there _was_ a breaking change, it's hard to say, just because there's nothing planned and it's hard to imagine what that would be so we'll have to play it by ear.

I think what's more likely is things like some new status codes (there will be at least 2 I think, for when threads are "full"), and there may be additional features, especially around moderation tooling, so that's what is likely to be added, I highly doubt any of that will be _breaking_ though.  But always a possibility.